src/main/java/frc/robot/subsystems/ball/shooter/ShooterImpl.java

```diff
@@ -50,8 +52,8 @@ private double getShooterRPM() {
     }
 
     private void setShooterSpeed(double speed) {
-        this.shooterController1.set(speed);
-        this.shooterController2.set(speed);
+        this.shooterController1.set(-speed);
+        this.shooterController2.set(-speed);
```

Comment:
Rather than just setting negative speeds here, could you invert the two controllers?